src/features/virtualized-table-renderers/lastModifiedByCellRenderer.js

```diff
@@ -9,13 +9,24 @@ type LastModifiedByCellRendererSettings = {
     dateFormat?: Object,
 };
 
-const lastModifiedByCellRenderer = (intl: IntlShape, { dateFormat }: LastModifiedByCellRendererSettings = {}) => (
+const lastModifiedByCellRenderer = (intl: any, { dateFormat }: LastModifiedByCellRendererSettings = {}) => (
     cellRendererParams: LastModifiedByCellRendererParams,
 ) =>
     baseCellRenderer(cellRendererParams, ({ modified_at, modified_by }: LastModifiedByCellRendererCellData) => {
-        const lastModified = dateFormat
-            ? intl.formatDate(modified_at, dateFormat)
-            : intl.formatRelative(Date.parse(modified_at), { units: 'day-short', style: 'numeric' });
+        let lastModified = '';
+
+        if (dateFormat) {
```

Comment:
Uses new version if available, otherwise old version.
Need to double check if the output is correct, specially the delta.